cinema/views.py

```diff
@@ -21,24 +26,43 @@
 )
 
 
-class GenreViewSet(viewsets.ModelViewSet):
+class GenreViewSet(
+    mixins.ListModelMixin, mixins.CreateModelMixin, viewsets.GenericViewSet
+):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
```

Comment:
to avoid code duplication add TokenAuthentication to the default authentication_classes in settings.py and add IsAdminOrIfAuthenticatedReadOnly to the default permission_classes in settings.py

Reply:
I've added it and removed the code duplication, but now the tests failed. I don't understand where the problem is
![image](https://github.com/user-attachments/assets/c28b2146-49ca-4806-a2c8-882da64e0426)